test/unit/proxy_api/proxy_exception_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class ProxyApiExceptionTest < ActiveSupport::TestCase
+  test 'exception response body is added if available' do
+    fake_exception = OpenStruct.new(:response => 'useful message from proxy',
+                                    :code => 400)
+    proxy_exception = ProxyAPI::ProxyException.new(
+      'fakeurl',
+      OpenStruct.new(:message => 'some error'),
```

Comment:
Shouldn't this second parameter be the `fake_exception`, rather than using `expects(:exception)` below?